Cargo.toml

```diff
@@ -34,6 +33,7 @@ url = { version = "2", features = ["serde"] }
 uuid = { version = "0.8", features = ["serde", "v4"] }
 ws = { version = "0.9", features = ["ssl"] }
 zeroize = "1"
+ethers-core = "0.6"
```

Comment:
I just checked and in [`ethers-core`](https://docs.rs/ethers-core/latest/src/ethers_core/types/mod.rs.html#8):
```rust
pub use ethabi::ethereum_types::{Address, Bloom, H160, H256, H512, U128, U256, U64};
```

So the type comes from [`ethabi`](https://docs.rs/ethabi/latest/src/ethabi/lib.rs.html#58):
```rust
pub use ethereum_types;
```

Which is using [`v0.13.1` of `ethereum_types`](https://github.com/rust-ethereum/ethabi/blob/7edf185125a9560d43bba0f84432a7cc874a5218/ethabi/Cargo.toml#L20):
```toml
ethereum-types = { version = "0.13.1", default-features = false }
```

So, really, I think that the easiest way to make it so you don't need to `mem::transmute` everywhere is to bump `ethereum-types` to `v0.13` instead of pulling in an additional dependency.

Sorry, I should have checked this earlier. I am super unfamiliar with the `ethers` crate and assumed it was using its own types.

Reply:
Hm... we still need the type `Bytes` which is defined in `ethers-core`, I don't see any equivalent in `ethereum-types`. What's your suggestion?

Reply:
You can use the H520 type for the signature from the `ethereum-types` crate right?

Reply:
Don't think so, it's need to be `0x` aware and able to ser/deser from string. which is what `Bytes` does.

Reply:
I tried it out locally and it seams to work:
```rust
    assert_eq!(
        serde_json::to_string(&ethereum_types::H520([66; 65])).unwrap(),
        "\"0x4242424242424242424242424242424242424242424242424242424242424242\
           4242424242424242424242424242424242424242424242424242424242424242\
           42\"",
    );
```

Reply:
What about the deserialization case? IIRC, I think I tried and didn't work out of the box.

Regardless, the response from `sign_transaction` is not the signature (H520) but the full raw tx which is +110 bytes.

We can stick to `String` if you want?

Reply:
No, it's fine as is then. Sorry for all the questions 🙈. I think ideally we wouldn't need this dependency (as you mentioned would require extracting the `Bytes` type somehow), but that can be done as a separate PR/commit.

Reply:
No worries & thanks for the merge. Can you deploy this version to cargo as well?